framework/src/media/Decoder.cpp

```diff
@@ -102,10 +116,10 @@ bool Decoder::empty()
 	return false;
 }
 
-size_t Decoder::getDataSpace()
+size_t Decoder::getAvailSpace()
 {
 #ifdef CONFIG_AUDIO_CODEC
-	return pv_player_dataspace(&mPlayer);
+	return rb_avail(mPlayer.rbsp->rbp);
```

Comment:
we have to use player api

Reply:
OK. I'll request to SRC-N. and will modify.